### ISSUE TITLE
Number of art in Collection of an address change to 0 when user delete the same view-only address

### ISSUE THREAD

**Issue by longbmk** [bitmark-inc/autonomy-apps#2833]
Steps:
1. Import a view-only address
2. Import the seeds of the above view-only address to have the full address
3. Observe the collection
4. Remove the view-only address at step #1
5. Observe the Collection page and the full address.

Observed result:
The artwork of full address change to 0 and collection does not show artwork of this full address

Expected result:
The collection should refresh to show artwork of the remaining not delete address.

https://github.com/bitmark-inc/autonomy-apps/assets/109057362/efa273d7-a80c-4098-9b37-ba0bfee1a59d

**Comment by phuocbitmark** [comment]
the best way to fix this problem is not allow user have 2 address but the same. to apply that, if user import (via seeds) an address that is the same as a view-only address they already have, it would replaces the view-only address; if users try to import a view-only address, we check if they already have that, if they have, we won't allow that
@bencebitmark we need your opinion and may be a design for this

**Comment by bencebitmark** [comment]
@phuocbitmark yeah i think you're right. i can't think of a use case where you'd need the address twice (view-only and seed import). 

we can probably change from view-only to imported address in this case.

**Comment by phuocbitmark** [comment]
@bencebitmark i need some design for that, when users import a address they already have, we need to notify them that we will replace the address or do not allow the import

**Comment by longbmk** [comment]
This issue no longer happens

**Comment by phuocbitmark** [comment]
[0.5 unit] not allow import twice
